modules/vba_documents/spec/request/metadata_request_spec.rb

```diff
@@ -92,54 +92,54 @@
         expect(upstream_service['details']['status']).to eq('Unavailable')
         expect(upstream_service['details']['time']).to eq('2020-09-21T00:00:00Z')
       end
+    end
+
+    context 'v1' do
```

Comment:
this just shifts v1 context out next to v0, instead of inside it for some strange reason